cate/ds/__init__.py

```diff
@@ -63,19 +64,24 @@ def cate_init():
             store_configs = yaml.safe_load(fp)
 
     for store_name, store_config in store_configs.items():
-        if store_config.get('store_id', '') == 'file' \
+        store_id = store_config.get('store_id')
+        assert_given(store_id, name='store_id', exception_type=RuntimeError)
```

Comment:
That's harsh. I think it would be enough to give out a warning and skip the store config.

Reply:
I don't agree. The store configuratuion is essential for Cate and a missing `store_id` is not allowed. Hence, Cate must not work with such a wrong configuration.

Reply:
Ideally, the JSON/YAML validation should take place after reading.